### PR TITLE
fix: 流水线插件执行快照页面插件入参渲染不出来问题 #6431

### PR DESCRIPTION
**Pull request by yjieliang**
fix: 流水线插件执行快照页面插件入参渲染不出来问题 #6431

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-ci?pullRequest=6627) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-ci?pullRequest=6627) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-ci?pullRequest=6627) it.</sub>

File: src/backend/ci/core/store/biz-store/src/main/kotlin/com/tencent/devops/store/resources/atom/UserAtomResourceImpl.kt [hunk 1]
```diff
@@ -46,7 +46,12 @@ class UserAtomResourceImpl @Autowired constructor(private val atomService: AtomS
     UserAtomResource {
 
     override fun getPipelineAtom(projectCode: String, atomCode: String, version: String): Result<PipelineAtom?> {
-        return atomService.getPipelineAtom(projectCode, atomCode, version)
+        return atomService.getPipelineAtom(
```
Comment (carlyin0801) [inline src/backend/ci/core/store/biz-store/src/main/kotlin/com/tencent/devops/store/resources/atom/UserAtomResourceImpl.kt:49]: 这个是让前端去传，你现在这样改可能会影响之前的业务逻辑的

File: src/backend/ci/core/store/biz-store/src/main/kotlin/com/tencent/devops/store/service/atom/impl/AtomServiceImpl.kt [hunk 2]
```diff
@@ -343,6 +343,7 @@ abstract class AtomServiceImpl @Autowired constructor() : AtomService {
         }
         if (queryOfflineFlag) {
             atomStatusList?.add(AtomStatusEnum.UNDERCARRIAGED.status.toByte()) // 也要给那些还在使用已下架的插件插件展示详情
+            atomStatusList?.add(AtomStatusEnum.GROUNDING_SUSPENSION.status.toByte()) // 存在测试插件取消发布的情形
```
Comment (carlyin0801) [inline src/backend/ci/core/store/biz-store/src/main/kotlin/com/tencent/devops/store/service/atom/impl/AtomServiceImpl.kt:346]: 处于上架中止的版本为什么要查出来呢

File: src/backend/ci/core/process/biz-base/src/main/kotlin/com/tencent/devops/process/engine/service/detail/TaskBuildDetailService.kt [hunk 3]
```diff
@@ -241,6 +233,7 @@ class TaskBuildDetailService(
         buildId: String,
         taskId: String,
         buildStatus: BuildStatus,
+        taskVersion: String? = null,
```
Comment (carlyin0801) [inline src/backend/ci/core/process/biz-base/src/main/kotlin/com/tencent/devops/process/engine/service/detail/TaskBuildDetailService.kt:236]: 传值逻辑没有看到

**Pull request closed without merging**